cli/cmd/testdata/install_addon.golden

```diff
@@ -3479,6 +3482,23 @@ spec:
           medium: Memory
         name: linkerd-identity-end-entity
 ---
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  labels:
+    app.kubernetes.io/name: prometheus
+    app.kubernetes.io/part-of: Linkerd
+    app.kubernetes.io/version: install-control-plane-version
+    linkerd.io/control-plane-component: prometheus
+  name: linkerd-prometheus
+  namespace: linkerd
+spec:
+  accessModes:
+    - 
+  resources:
+    requests:
+      storage: 
+---
```

Comment:
We can't have empty resources like this, so I think we'll just have to comment-out all the entries under `persistence` in `charts/add-ons/prometheus/values.yaml`, as they originally were in `charts/linkerd2/values.yaml`

Reply:
I see! We'll have to revisit those if we can get helm-docs to support values in comments.